cmd/grpcurl/grpcurl.go

```diff
@@ -615,7 +619,12 @@ func main() {
 				// for messages, also show a template in JSON, to make it easier to
 				// create a request to invoke an RPC
 				tmpl := grpcurl.MakeTemplate(dsc)
-				_, formatter, err := grpcurl.RequestParserAndFormatterFor(grpcurl.Format(*format), descSource, true, false, nil)
+				options := grpcurl.FormatterOptions{
+					EmitJSONDefaultFields: true,
+					IncludeTextSeparator:  false,
+					AllowUnknownFields:    *allowUnknownFields,
```

Comment:
Let's just hardcode to `false`, since this code path won't be parsing anything.

Also, you can just omit the literal false values since they are default/zero vals:

```
options := grpcurl.FormatOptions{EmitJSONDefaultFields: true}
```